lib/openshift/cluster_log_forwarder.rb

```diff
@@ -1,8 +1,8 @@
 # represent a clusterlogforwarder object
-require 'openshift/cluster_resource'
+require 'openshift/project_resource'
 
 module BushSlicer
-  class ClusterLogForwarder < ClusterResource
+  class ClusterLogForwarder < ProjectResource
```

Comment:
Seems I can not find this resource type.
```
$ oc version
Client Version: 4.6.0-202008260718.p0-931d20f
Server Version: 4.6.0-rc.2
Kubernetes Version: v1.19.0+d59ce34

$ oc api-resources | grep -E -i 'log|NAME'
NAME                                  SHORTNAMES       APIGROUP                              NAMESPACED   KIND
namespaces                            ns                                                     false        Namespace
consoleexternalloglinks                                console.openshift.io                  false        ConsoleExternalLogLink
netnamespaces                                          network.openshift.io                  false        NetNamespace
catalogsources                        catsrc           operators.coreos.com                  true         CatalogSource

```

Reply:
@liangxia you can only find it after deploying logging 4.6

Reply:
```
$ oc api-resources |grep logging
clusterlogforwarders                  clf              logging.openshift.io                  true         ClusterLogForwarder
clusterloggings                       cl               logging.openshift.io                  true         ClusterLogging
elasticsearches                       es               logging.openshift.io                  true         Elasticsearch
kibanas                                                logging.openshift.io                  true         Kibana
```
Logging operators are not deployed along with the OCP.

Reply:
Ack